fetch.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
```

Comment:
I'm thinking the `fetch.sh` file should not be included in this repo. We should instead have a README.md related to the `create_test_users_and_applications_batch.py` command with instructions that reference the BFD's documentation for synthetic data and how to copy the needed beneficiary RIF files it in to place. This way we would not need to keep an updated version in this repo.  BFD's related doc:  https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-model/bfd-model-rif-samples/dev/design-sample-data-sets.md